### ISSUE TITLE
Cannot tab into any language or logout option

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#373]
When using a screen reader cannot tab into language or logout option and so cannot be selected
![screen shot 2016-07-06 at 14 34 36](https://cloud.githubusercontent.com/assets/13200338/16619672/00fd1c24-4387-11e6-8823-a786d6b6547d.png)

**Comment by DenisVuyka** [comment]
Do we need a demo application support that? This language picker is an example to demo i18n support.

**Comment by hashraf88** [comment]
Possibly not, @magemello do we need to support this?
